JeMPI_Apps/JeMPI_UI/src/components/dashboard/widgets/ConfusionMatrixWidget.tsx

```diff
@@ -0,0 +1,198 @@
+import { Box, Divider, Grid, Typography } from '@mui/material'
+
+const ConfusionMatrix = () => {
+  return (
+    <>
+      <Grid container padding={1} minHeight={'450px'}>
+        <Grid
+          item
+          lg={6}
+          xs={12}
+          sx={{
+            backgroundColor: '#D5E8D4',
+            borderBottomLeftRadius: { lg: '2rem' },
+            borderTopLeftRadius: { lg: '2rem' },
+            textAlign: 'center',
+            boxShadow: 3
+          }}
+        >
+          <Box sx={{ padding: '1rem' }}>
+            <Typography variant="h5">False Negatives</Typography>
+            <Typography variant="h3">50</Typography>
+          </Box>
+          <Box sx={{ display: 'flex', justifyContent: 'end' }}>
+            <Box
+              sx={{
+                width: '50%',
+                padding: '1rem',
+                border: '1px solid #9FD383',
+                backgroundColor: '#9FD383',
+                borderRadius: '1rem 0rem 0rem 1rem',
+                boxShadow: 3
+              }}
+            >
+              <Typography variant="h5" color={'#fff'}>
+                True Positives
+              </Typography>
+
+              <Typography variant="h3" color={'#fff'}>
+                {(5000).toLocaleString()}
```

Comment:
Is there any particular reason you are using `toLocaleString` here but not for 'False Negative' above?